.jenkins/lsu/slurm-configuration-gcc-9-cuda-11.sh

```diff
@@ -4,5 +4,5 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-configuration_slurm_partition="sc15"
+configuration_slurm_nodes="geev"
```

Comment:
Do we need this *if* we can compile for multiple architectures at the same time? It'd be nice to keep two nodes for CUDA builds if possible.

Reply:
not at all that's why I called the commit [Tmp] I was just trying it out as I couldn't get a manual allocation on rostam